modules/util/source-formatter/src/main/resources/dependencies/replacements.json

```diff
@@ -3166,6 +3166,14 @@
 		"skipParametersValidation": true,
 		"to": "CPTaxCategoryCreateDateComparator.getInstance(param#0#)"
 	},
+	{
+		"classNames": [
+			"IndexWriterHelper"
+		],
+		"from": "updateDocument(String paramName, long paramName, Document paramName, boolean paramName)",
+		"issueKey": "LPD-35875",
+		"to": "updateDocument(param#1#, param#2#)"
+	},
```

Comment:
@MicaelleSilvaa  https://github.com/liferay-devtools/liferay-portal/blob/8e9196dc6656c42f6cc2514e87dbdb88a883344c/modules/util/source-formatter/src/main/resources/dependencies/replacements.json#L50

The `replacements.json` already has a pattern for a similar case in the utility class, can you check if doesn't also happen in the utility? If so, add in the `classNames` attribute.